iceoryx_posh/include/iceoryx_posh/popo/untyped_publisher.hpp

```diff
@@ -59,6 +59,14 @@ class UntypedPublisherImpl : public base_publisher_t
     ///
     void publish(const void* chunk) noexcept;
 
+    ///
+    /// @brief Releases the chunk provided by its payload pointer.
```

Comment:
maybe add that the ownership is released and that the chunk must not be accessed afterwards